charts/cryostat/templates/pvc.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.pvc.enabled }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Release.Name }}
```

Comment:
I noticed other templates are using `{{ include "cryostat.fullname" . }}` for their name instead of `{{ .Release.Name }}`. I think in most cases they're the same, but they may not be. We should probably do the same here. Sorry about that, I forgot all about `fullname`.